------ a/101worker/tools/reporter.py
+++ b/101worker/tools/reporter.py
@@ -9,28 +9,28 @@ from mailer import Message

 # Configure addresses and server
 emailFrom = "softlang@uni-koblenz.de"
 #emailTo = "dotnetby@gmail.com" # for testing
-emailTo = "freddy92.online@gmail.com"
+emailTo = "101companies@gmail.com"
 emailServer = "deliver.uni-koblenz.de"

 tmpl = "<table><tr><td>Status</td><td>Module</td></tr>"
 tmpl_end = "</table>"

 def buildMailContent():
-    #logfile = open('../101logs/runner.log', 'r')
-    #lastArchive = open('../101web/logs/lastArchive', 'r')
-    #time = lastArchive.read()
+	logfile = open('../101logs/runner.log', 'r')
+	lastArchive = open('../101web/logs/lastArchive', 'r')
+	time = lastArchive.read()
 	log = csv.reader(open('../101logs/runner.log', 'r'), delimiter=';', quotechar='|')
 	content = tmpl
 	for row in log:
 		if len(row) > 2:
 			l = "<tr><td>"+row[3]+"</td><td><a href='http://worker.101companies.org/logs/"+time+"/"+row[0]+".log'>"+row[0]+"</a></td></tr>"
-			print l
+			#print l
 			content += l
 	content += tmpl_end
 	return content

 # Send the email
-#log = buildMailContent()
+log = buildMailContent()
 #if (string.find(log, 'FAIL')>=0):
 	#msg = MIMEText(log.encode('utf-8'), 'text/html', 'utf-8')
 	#msg["To"] = emailTo
@@ -42,6 +42,6 @@ def buildMailContent():

 x = 2
 message = Message(From=emailFrom,To=emailTo)
 message.Subject = "[101worker] Execution Status Report"
-#message.Html = log
+message.Html = log
 sender = Mailer(emailServer)
 sender.send(message)
